tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
```

Comment:
remove the commented-out lines for simplicity?

Reply:
I think ill keep these as its generated by the ts init config. We may want to update in the future.